tldw_Server_API/app/services/admin_webhooks_service.py

```
@@ -0,0 +1,442 @@
+            count_row = await pool.fetchone(
+                "SELECT COUNT(*) as cnt FROM admin_webhooks",
+            )
+            rows = await pool.fetchall(
+                "SELECT * FROM admin_webhooks ORDER BY created_at DESC LIMIT ? OFFSET ?",
+                (limit, offset),
+            )
+        total = count_row["cnt"] if count_row else 0
+        return [self._row_to_record(r) for r in rows], total
+
+    async def update_webhook(
+        self, webhook_id: int, **fields: Any,
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**fields`

(ANN401)